tests/utils.py

```diff
@@ -0,0 +1,38 @@
+"""Utility functions for testing."""
+from contextlib import contextmanager
+from typing import Type, Union
+
+from flexmock._api import RE_TYPE
+
+
+@contextmanager
+def assert_raises(expected_exception: Type[BaseException], match: Union[RE_TYPE, str, None]):
```

Comment:
You know, it's minor, but if we switched the arguments we could write this cleaner with [singledispatch](https://docs.python.org/3/library/functools.html#functools.singledispatch), though I don't know  how nuch sense the signature would make :thinking: (unless we currify ? Am I thinking too much ? Probably.)

Reply:
Yeah, I agree that you are thinking too much :smile: